packages/pass-style/src/passStyleOf.js

```diff
@@ -177,7 +182,7 @@ const makePassStyleOf = passStyleHelpers => {
           return 'remotable';
         }
         case 'function': {
-          isFrozen(inner) ||
+          isNoTrapping(inner) ||
             Fail`Cannot pass non-frozen objects like ${inner}. Use harden()`;
```

Comment:
Changing to

```suggestion
            Fail`Cannot pass trapping objects like ${inner}. Use harden()`;
```

would technically be more correct. But will less often be understood.